### PR TITLE
Feature: pixel color

### PR DESCRIPTION
**Pull request by vadymmarkov**
There is a new subscript on `UIImage` to return color at the point (x, y).

``` swift
image[0, 0]
```

### CODE REVIEW

File: Source/iOS/UIImage+Hue.swift [hunk 1]
```diff
@@ -146,4 +146,22 @@ extension UIImage {
       secondaryColor ?? (isDarkBackgound ? whiteColor : blackColor),
       detailColor    ?? (isDarkBackgound ? whiteColor : blackColor))
   }
+
+  public subscript (x: Int, y: Int) -> UIColor? {
```
Comment (zenangst) [inline Source/iOS/UIImage+Hue.swift]: Why subscripting instead of a function? 😁

Comment (vadymmarkov) [reply Source/iOS/UIImage+Hue.swift]: I don't know, but I think I agree that function is better for autocompletion. Will change a bit later today.

**Comment by vadymmarkov** [comment]
@zenangst Here you go.

File: Source/iOS/UIImage+Hue.swift [hunk 2]
```diff
@@ -146,4 +146,22 @@ extension UIImage {
       secondaryColor ?? (isDarkBackgound ? whiteColor : blackColor),
       detailColor    ?? (isDarkBackgound ? whiteColor : blackColor))
   }
+
+  public func color(at point: CGPoint) -> UIColor? {
+    guard point.x >= 0 && point.x <= size.width
```
Comment (onmyway133) [inline Source/iOS/UIImage+Hue.swift]: Can we use `CGRectContainsPoint` ?

**Comment by onmyway133** [comment]
🎉

**Comment by zenangst** [comment]
Sweet! 😁

**Pull request merged**